### PR TITLE
docs: replace broken bundle analyzer link

### LINKED ISSUE #68651: Docs: Broken link on Memory Usage
### What is the update you wish to see?

The link should lead to here:
[Bundle Analyzer](https://nextjs.org/docs/app/building-your-application/optimizing/bundle-analyzer)

### Is there any context that might help us understand?

Bundle Analyzer link on the [Memory usage](https://nextjs.org/docs/app/building-your-application/optimizing/memory-usage) page's [Reduce number of dependencies](https://nextjs.org/docs/app/building-your-application/optimizing/memory-usage#reduce-number-of-dependencies) section leads to a broken link: https://nextjs.org/docs/app/building-your-application/optimizing/package-bundling#analyzing-javascript-bundles

Screenshot 2024-08-08 at 09 36 07

### Does the docs page already exist? Please link to it.

https://nextjs.org/docs/app/building-your-application/optimizing/memory-usage

### PR DESCRIPTION
**Pull request by mehmetozguldev**
### What?
[Memory Usage](https://nextjs.org/docs/app/building-your-application/optimizing/memory-usage) page has a broken link, leading to: [Package Bundling](https://nextjs.org/docs/app/building-your-application/optimizing/package-bundling#analyzing-javascript-bundles) (Page not found)

### How?
I replaced the link on the markdown file with the correct page and the correct section: [Bundle Analyzer - Analyzing your bundles](https://nextjs.org/docs/app/building-your-application/optimizing/bundle-analyzer#analyzing-your-bundles)

Fixes #68651

### CODE REVIEW

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: 996696664c7289c1bf7ecffbd79431ff02dc4c98

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

**Review by samcx — APPROVED** [review]
Thank you for submitting a PR!

<picture data-single-emoji=":lgtm:" title=":lgtm:">:lgtm:</picture>

**Comment by samcx** [comment]
@mehmetozguldev Just <picture data-single-emoji=":fyi:" title=":fyi:">:fyi:</picture>, when your PR merges, it should be present soon in https://rc.nextjs.org/docs, but not immediately. :ty-dark:

**Comment by mehmetozguldev** [comment]
> @mehmetozguldev Just <picture data-single-emoji=":fyi:" title=":fyi:">:fyi:</picture>, when your PR merges, it should be present soon in https://rc.nextjs.org/docs, but not immediately.

Great, thanks for the updates!

File: docs/02-app/01-building-your-application/06-optimizing/13-memory-usage.mdx [hunk 1]
```diff
@@ -11,7 +11,7 @@ Let's explore some strategies and techniques to optimize memory and address comm
 
 Applications with a large amount of dependencies will use more memory.
 
-The [Bundle Analyzer](/docs/app/building-your-application/optimizing/package-bundling#analyzing-javascript-bundles) can help you investigate large dependencies in your application that may be able to be removed to improve performance and memory usage.
+The [Bundle Analyzer](/docs/app/building-your-application/optimizing/bundle-analyzer#) can help you investigate large dependencies in your application that may be able to be removed to improve performance and memory usage.
```
Comment (delbaoliveira) [inline docs/02-app/01-building-your-application/06-optimizing/13-memory-usage.mdx:14]:
```suggestion
The [Bundle Analyzer](/docs/app/building-your-application/optimizing/bundle-analyzer) can help you investigate large dependencies in your application that may be able to be removed to improve performance and memory usage.
```

**Review by delbaoliveira — APPROVED** [review]

File: docs/02-app/01-building-your-application/06-optimizing/13-memory-usage.mdx [hunk 2]
```diff
@@ -11,7 +11,7 @@ Let's explore some strategies and techniques to optimize memory and address comm
 
 Applications with a large amount of dependencies will use more memory.
 
-The [Bundle Analyzer](/docs/app/building-your-application/optimizing/package-bundling#analyzing-javascript-bundles) can help you investigate large dependencies in your application that may be able to be removed to improve performance and memory usage.
+The [Bundle Analyzer](/docs/app/building-your-application/optimizing/bundle-analyzer) can help you investigate large dependencies in your application that may be able to be removed to improve performance and memory usage.
```
Comment (delbaoliveira) [inline docs/02-app/01-building-your-application/06-optimizing/13-memory-usage.mdx:14]:
```suggestion
The [Bundle Analyzer](/docs/app/building-your-application/optimizing/package-bundling) can help you investigate large dependencies in your application that may be able to be removed to improve performance and memory usage.
```

**Comment by samcx** [comment]
@mehmetozguldev One extra new tidbit—you can just access the regular docs instead (rc.nextjs.org will just redirect to nextjs.org)

**Pull request merged**